VideoApp/VideoApp/SwiftUI/Screens/Room/RoomViewModel.swift

```diff
@@ -40,6 +40,10 @@ import Combine
     @Published var isShowingStats = false
     @Published var isShowingCaptions = false {
         didSet {
+            guard captionsManager != nil else {
+                return /// When not configured do nothing so preview doesn't crash
+            }
```

Comment:
This is kind of clunky but for now I stuck with the status quo for preview configuration. I could maybe see using optionals or some other better way down the road.